components/constellation/sandboxing.rs

```diff
@@ -128,7 +129,7 @@ pub fn content_process_sandbox_profile() -> Profile {
     all(target_arch = "aarch64", not(target_os = "macos"))
 ))]
 pub fn content_process_sandbox_profile() {
-    error!("Sandboxed multiprocess is not supported on this platform.");
+    log::error!("Sandboxed multiprocess is not supported on this platform.");
```

Comment:
I would import `use log::error` for consistency with the rest of the code here and below,

Reply:
Because there are cfgs on linux we would get warnings for unused code.